src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -11856,7 +11853,28 @@ void VRFCondenserEquipment::CalcVRFCondenser_FluidTCtrl(EnergyPlusData &state)
                                                    CapMinTe + 8,
                                                    this->IUCondensingTemp - 5);
 
-        if ((Q_c_OU * C_cap_operation) <= CompEvaporatingCAPSpdMin) {
+        Real64 CompEvaporatingCAPSpdMaxCurrentTsuc = this->CoffEvapCap * this->RatedEvapCapacity *
+                                                     CurveValue(state, this->OUCoolingCAPFT(NumOfCompSpdInput), Tdischarge, this->EvaporatingTemp);
+        Real64 CompEvaporatingPWRSpdMaxCurrentTsuc =
+            this->RatedCompPower * CurveValue(state, this->OUCoolingPWRFT(NumOfCompSpdInput), Tdischarge, this->EvaporatingTemp);
+        if (CompEvaporatingCAPSpdMin > CompEvaporatingCAPSpdMaxCurrentTsuc) {
+            ShowSevereMessage(state, format("{} \"{}\":", cVRFTypes(VRF_HeatPump), this->Name));
+            ShowContinueError(state,
+                              format(" Evaporative Capacity at max speed is smaller than evaporative capacity at min speed, "
+                                     "{:.3T} < {:.3T}",
+                                     CompEvaporatingCAPSpdMaxCurrentTsuc,
+                                     CompEvaporatingCAPSpdMin));
+        }
```

Comment:
Should add the recurring error format so this warning, however unlikely, does not fill the err file if it ever does report. Also, if this warning does report shouldn't the model use one of these capacities (CAPSpdMin?) to set other variables so that subsequent calculations can proceed?

Reply:
add the recurring error format in commit a2a0441395 "Fix recurring error"